staging/src/k8s.io/apiserver/pkg/server/hooks.go

```diff
@@ -151,15 +157,16 @@ func (s *GenericAPIServer) AddPreShutdownHookOrDie(name string, hook PreShutdown
 	}
 }
 
-// RunPostStartHooks runs the PostStartHooks for the server
-func (s *GenericAPIServer) RunPostStartHooks(stopCh <-chan struct{}) {
```

Comment:
I checked with grep.app and only found calls in the apiserver or forks of it.

If you think that we should keep it, then I can keep it as before and add RunPostStartHooksWithContext.`

Reply:
yeah, looks like it could have been a private function of `GenericAPIServer`. I don't think we need `RunPostStartHooksWithContext`. @sttts thoughts?

Reply:
Changing the signature should be fine. I don't think we have to be too careful for server functionality (versus client-go and friends).

> yeah, looks like it could have been a private function of GenericAPIServer

Not sure I can follow. This function (whether with context or channel) is part of the public interface of k8s.io/apiserver, and used in projects (I know heavy use in kcp for example).